libs/category/state/src/facades/category.facade.spec.ts

```diff
@@ -345,4 +351,12 @@ describe('DaffCategoryFacade', () => {
       expect(facade.isCategoryEmpty$).toBeObservable(expected);
     });
   });
+
+  describe('isPageMutating$', () => {
+
+    it('should be false', () => {
```

Comment:
even though I don't think we need to retest everything in the facades that is tested in the selectors, I still think this should be "should return whether the category page is mutating" with `const expected = cold('a', { a: categoryPageConfigurationState.daffState === DaffState.Mutating });`

Reply:
changed